NotificationApi/NotificationApi.DAL/Exceptions/DuplicateNotificationTemplateException.cs

```diff
@@ -2,18 +2,20 @@
 using System.Runtime.Serialization;
 using NotificationApi.Domain.Enums;
 
-namespace NotificationApi.DAL.Exceptions;
-
-[Serializable]
-public class DuplicateNotificationTemplateException : Exception
+namespace NotificationApi.DAL.Exceptions
```

Comment:
the compiler was throwing warnings about not being within named namespaces so I added the curly braces to reduce the noise from builds